lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -774,7 +774,16 @@ handle_heredocs(T, Line, Column, H, Scope, Tokens) ->
 handle_strings(T, Line, Column, H, Scope, Tokens) ->
   case elixir_interpolation:extract(Line, Column, Scope, true, T, H) of
     {error, Reason} ->
-      interpolation_error(Reason, [H | T], Scope, Tokens, " (for string starting at line ~B)", [Line]);
+      {error, {Meta, Message, Token}, NewRest, NewScope, NewTokens} = interpolation_error(Reason, [H | T], Scope, Tokens, " (for string starting at line ~B)", [Line]),
+      NewMeta = case lists:keyfind(error_type, 1, Meta) of
+        % Don't override other errors that may have happened downstream already
+        {error_type, mismatched_delimiter} -> Meta;
+        {error_type, invalid_bidi} -> lists:keydelete(error_type, 1, Meta);
+        _ ->
+          {line, EndLine} = lists:keyfind(line, 1, Meta),
+          [{line, Line}, {column, Column - 1}, {error_type, unclosed_delimiter}, {end_line, EndLine}, {opening_delimiter, '"'}]
```

Comment:
The current column position at this state was always 1 ahead of the start delimiter
I'm not sure if this is related to https://github.com/elixir-lang/elixir/issues/13184, but adding a -1 solved it for this case
I haven't dived deep into this, but we might want to check what's going on here

Reply:
This is happening in this case because we inject one additional character when parsing the interpolation. You can see that in the success clause we add `Column - 1` too to get the proper offset.